dynamic-layers/openembedded-layer/recipes-devtools/python3-adafruit-circuitpython-busdevice_5.0.5.bb

```diff
@@ -16,3 +16,4 @@ RDEPENDS:${PN} += " \
     python3-adafruit-blinka \
     python3-core \
 "
+COMPATIBLE_HOST:libc-musl:class-target = "null"
```

Comment:
Is it also the case for the other python projects or only for "blinka" as the commit log says?

Reply:
Others need blinka to build so they are using it as direct dependency which won’t be available

Reply:
Makes sense.